### PR TITLE
Market 6824 Fixed the Retail Sample App

### PR DESCRIPTION
**Pull request by cs185388**
Reach out to me with any questions/if you need details of what has changed

### CODE REVIEW

File: components/admin/groups/Groups.js [hunk 1]
```diff
@@ -20,14 +21,14 @@ function Groups({ data, isError, isLoading }) {
...
-              {groups.map((item) => (
+              {activeGroups.map((item) => (
                 <tr key={item.groupId.groupCode}>
                   <th scope="row">{item.groupId.groupCode}</th>
                   <td>{item.title.value}</td>
                   <td>{item.status}</td>
                   <td>
                     <a href={`/admin/groups/${item.groupId.groupCode}`}>
-                      <FontAwesomeIcon icon={faEdit} color="darkslategrey" />
+                      {/*<FontAwesomeIcon icon={faEdit} color="darkslategrey" />*/}
```
Comment (av185144) [inline components/admin/groups/Groups.js:31]: get rid of comments

File: pages/admin/orders.js [hunk 2]
```diff
@@ -1,25 +1,27 @@
-import React from 'react';
+import React, {useContext} from 'react';
 import useDashboard from '~/lib/swr/useDashboard';
 import Orders from '~/components/admin/orders/Orders';
 import Layout from '~/components/admin/Layout';
+import {UserStoreContext} from "~/context/userStore";
```
Comment (av185144) [inline pages/admin/orders.js:5]:
```suggestion
import { UserStoreContext } from "~/context/userStore";
```

**Review by av185144 — CHANGES_REQUESTED** [review]

File: components/admin/groups/Groups.js (hunk 1 above)
Comment (cs185388) [reply components/admin/groups/Groups.js:31]: I have left some commented code intentionally to indicate buttons that I have hidden and need to fixed later on. I'm open to removing that but we will have to take note of exactly what was removed and from where in the codebase.

File: components/public/user/ProfileForm.js [hunk 3]
```diff
@@ -127,13 +127,17 @@ export default function ProfileForm({ session, user, logs, setLogs }) {
               <Col>
                 <h4 className="mb-2">My Profile</h4>
               </Col>
+            {
+              /*
```
[the remark this answers is not in the document]
Comment (cs185388) [reply components/public/user/ProfileForm.js:131]: I have left some commented code intentionally to indicate buttons/actions that I have hidden and need to be fixed later on. I'm open to removing that but we will have to take note of exactly what was removed and from where in the codebase.

File: components/admin/groups/Groups.js (hunk 1 above)
Comment (av185144) [reply components/admin/groups/Groups.js:31]: yea just add it as a TODO comment

**Review by av185144 — APPROVED** [review]

File: components/public/sites/FindStoreModal.js [hunk 4]
```diff
@@ -32,6 +32,9 @@ const FindStoreModal = (props) => {
           .then((body) => {
             const { response } = body;
             setSites(response.data.pageContent);
+          })
+          .catch((e) => {
```
Comment (mc185335) [inline components/public/sites/FindStoreModal.js:36]: we want to keep this console.log?

**Review by mc185335 — COMMENTED** [review]
there's a few console logs for error handling do we want those?

File: components/public/sites/FindStoreModal.js (hunk 4 above)
Comment (cs185388) [reply components/public/sites/FindStoreModal.js:36]: I was thinking we keep some of these error console logs for now as a placeholder for the developer who picks up a future ticket for error handling and displaying messages

**Review by mc185335 — APPROVED** [review]

**Review by jg185352 — APPROVED** [review]

**Pull request merged**